lib/routes/account.js

```diff
@@ -877,26 +879,26 @@ module.exports = (
             && ! doSigninConfirmation
             && emailRecord.emailVerified
           if (shouldSendNewDeviceLoginEmail) {
-            return getGeoData(ip)
-              .then(
-                function (geoData) {
-                  mailer.sendNewDeviceLoginNotification(
-                    [],
-                    emailRecord,
-                    {
-                      acceptLanguage: request.app.acceptLanguage,
-                      flowId: flowId,
-                      flowBeginTime: flowBeginTime,
-                      ip: ip,
-                      location: geoData.location,
-                      timeZone: geoData.timeZone,
-                      uaBrowser: sessionToken.uaBrowser,
-                      uaBrowserVersion: sessionToken.uaBrowserVersion,
-                      uaOS: sessionToken.uaOS,
-                      uaOSVersion: sessionToken.uaOSVersion,
-                      uaDeviceType: sessionToken.uaDeviceType
-                    }
-                  )
+            const promises = [getGeoData(ip), features.isSecondaryEmailEnabled() ? db.accountEmails(sessionToken.uid) : P.resolve([])]
+            return P.all(promises)
+              .spread((geoData, emails) => {
+                mailer.sendNewDeviceLoginNotification(
```

Comment:
let's a doc here that this email is sent to `emails` and `emailRecord.email` (?) 

and that the message to `emails` gets sent as a CC